_data/students.yml

```diff
@@ -506,4 +506,11 @@
   blog: http://disam8853.github.io/
   twitter: Disam_Joker
   facebook: www.sam3
-  bio: I love everyone
\ No newline at end of file
+  bio: I love everyone
+
+- name: Marks Polakovs
+  github: marksomnian
+  image: markspolakovs.jpg
```

Comment:
*Error displaying picture*. Your image name is `MarksPolakovs.jpg` whereas you are trying to access a non-available `markspolakovs.jpg`